instructure/canvas-ios
# Offline course sync people list

refs: MBL-16766
affects: Student
release note: None.
test plan:
- Sync People
- Check listed people for multiple courses
- Check user role in the subtitles
- Tap on a list item should trigger the offline unavailability alert.
- Search should be disabled and trigger the same alert.

## Checklist
